app/javascript/mastodon/api/domain_blocks.ts

```diff
@@ -0,0 +1,13 @@
+import api, { getLinks } from 'mastodon/api';
+
+export const apiGetDomainBlocks = async (url?: string) => {
+  const response = await api().request<string[]>({
+    method: 'GET',
+    url: url ?? 'api/v1/domain_blocks',
```

Comment:
This will break in deck mode.
```suggestion
    url: url ?? '/api/v1/domain_blocks',
```